ape_safe/_cli.py

```diff
@@ -98,22 +98,37 @@ def remove(cli_ctx, alias):
     cls=NetworkBoundCommand, short_help="See pending transactions for a locally-tracked Safe"
 )
 @network_option()
+@click.option("sign_with_local_signers", "--sign", is_flag=True)
+@click.option("--execute", is_flag=False, flag_value=True, default=False)
 @existing_alias_argument(account_type=SafeAccount)
-def pending(network, alias):
+def pending(network, sign_with_local_signers, execute, alias):
     safe = accounts.load(alias)
-    local_signers = set(signer for signer in accounts if signer.address in safe.signers)
-    if local_signers:
-        click.echo("Local Signer(s) detected!")
-        sign_with_local_signers = click.confirm("Do you want to sign unconfirmed transactions")
+    submitter = None
 
-    else:
-        sign_with_local_signers = False
+    if isinstance(execute, str):
+        if execute in accounts.aliases:
+            submitter = accounts.load(execute)
+        elif execute in accounts:
+            submitter = accounts[execute]
+        else:
+            raise AccountsError(f"`--execute` value '{execute}` not found in local accounts.")
+
+    elif execute is True:
+        submitter = safe.local_signers[0]
```

Comment:
this needs to handle the case when there are not local signers or else youll get an IndexError here.
it probably just not allow execute if there are no local signers

something like:

```python
elif execute is True and len(self.local_signers) > 0:
    submitter = safe.local_signers[0]

elif execute is True:
   raise AccountsError("Cannot `--execute` without local signer.")